### PR TITLE
Bump cypress@12.6.0

### PR DESCRIPTION
**Pull request by FriggaHel**

### CODE REVIEW

File: v1/examples/webkit/.sauce/config.yml [hunk 1]
```diff
@@ -11,15 +11,15 @@ sauce:
...
+  version: 12.6.0
   configFile: "cypress.config.js"
 
 rootDir: ./
 
 suites:
   - name: (Experimental) webkit test
     browser: webkit
-    platformName: Windows 11
+    platformName: macOS 12
```
Comment (alexplischke) [inline v1/examples/webkit/.sauce/config.yml:22]: Is this change intentional? And if so, maybe we should test against both platforms here?

Comment (FriggaHel) [reply v1/examples/webkit/.sauce/config.yml:22]: Yes, to make the example work.
It appears that on macOS 11 webkit is crashing at startup, on Windows 11, it's timeouting (Cypress loads, but never goes after the `Loading your tests` step.
It's working fine on macOS 12. I moved to this to have working example while I'm troubleshooting this.

Also, I'll add a bigger scope regarding runner testing, to avoid surprises like this.

**Review by alexplischke — APPROVED** [review]

**Review by tianfeng92 — APPROVED** [review]
